cabal.project

```diff
@@ -204,10 +206,17 @@ source-repository-package
   tag: f73079303f663e028288f9f4a9e08bcca39a923e
   --sha256: 1n87i15x54s0cjkh3nsxs4r1x016cdw1fypwmr68936n3xxsjn6q
 
--- The r0 revision of this quickcheck-state-machine-0.7 on Hackage adds a lower
```

Comment:
As John Ky noted elsewhere, we no longer need this since we're only on GHC 8.10 now.